frontend-react/src/components/FileHandlers/FileHandlerErrorsWarningsStep.tsx

```diff
@@ -68,23 +88,27 @@ export const FileHandlerErrorsWarningsStep = ({
                     message={`The file does not meet the jurisdiction's schema. Please resolve the errors below.`}
```

Comment:
This can be a string.
"The file does not meet the jurisdiction's schema. Please resolve the errors below."